src/main/assets/js/main.js

```diff
@@ -26,7 +26,62 @@ angular.module('slicebox', [
     'slicebox.adminSystem'
 ])
 
-.config(function($locationProvider, $routeProvider, $mdThemingProvider, $filterProvider, $httpProvider) {
+.provider('sbxUtil', function() {
```

Comment:
This would normally be a factory but has to be a provider in order to be able to use it in the setting up of filters below. Just a AngularJS thing...